tools/cuttlefish-host-image-installer/update_github_releases.sh

```diff
@@ -20,6 +20,19 @@ echo "$changes" > changelog
 
 STABLE_VERSION="v$version"
 
+echo "step 2: move stable release to new commit id"
```

Comment:
What's the significance of having both `stable` and `latest` github releases if they're updated at the same time?

Reply:
Currently, they are the same. Originally, we want stable release stores stable version, and latest release stores latest version. But we want users to use stable release instead of ToT, therefore, the latest release still stores stable version.